src/components/dashboard/Dashboard.js

```diff
@@ -10,11 +10,30 @@ const Dashboard = () => {
   return (
     <div>
       <div className="dashboard_main"><h1 className="dashboard_header">Dashboard</h1></div>
-      <Scheduler />
-      <Progress />
-      <button style={{ marginTop: '50px' }} type='submit' className="dashboard_btn" onClick={() => history.push("/schedule")}>
-        Schedule
+      <div>
+      <button type='submit' className="dashboard_btn" onClick={() => history.push("/")}>
+        Create a custom workout plan
       </button>
+      <button type='submit' className="dashboard_btn" onClick={() => history.push("/")}>
+        Find a prefab plan
+      </button>
+      </div>
+      <div>
+      <button type='submit' className="dashboard_btn" onClick={() => history.push("/")}>
+        Create custom meal plan
+      </button>
+      <button type='submit' className="dashboard_btn" onClick={() => history.push("/")}>
+        Find a prefab meal plan
```

Comment:
@Rrawla2 Is there supposed to be two buttons for Find a prefab meal plan?

Reply:
Only 1.  I can rename the other to prefab workout plan instead of prefab plan

Reply:
Oh I see. I got confused. Yeah. I think renaming the other to prefab workout plan would be more distinguishable. Thank you!

Reply:
Ok fixed the button text and pushed.  Thanks for catching that mistake!

Reply:
Thank you! :)